### PR TITLE
Added default reviewer notes, made field mandatory

### PR DESCRIPTION
**Pull request by ibroom**
Untested, please confirm this is mandatory and renders correctly on add/edit and changes are saved correctly.

@jgustafsson24 cc

### CODE REVIEW

File: interface.html [hunk 1]
```diff
@@ -1051,7 +1051,24 @@ <h3><small>Reviewer Notes</small></h3>
                     <p class="help-block label-helper"><small>Optional</small></p>
                   </div>
                   <div class="col-sm-8">
-                    <textarea name="fl-store-revNotes" class="form-control" id="fl-store-revNotes" rows="10"></textarea>
+                    <textarea name="fl-store-revNotes" class="form-control" id="fl-store-revNotes" rows="10" required>
```
Comment (squallstar) [inline interface.html]: @ibroom why did you add the `required`?

Comment (ibroom) [reply interface.html]: The field should be mandatory.

Comment (squallstar) [reply interface.html]: I understand but changing it on the code will break enterprise submissions (since the field is not required). I believe the "required" functionality will have to be dealt with on the JS code of the component, @hcarneiro correct me if I'm wrong. 

@ibroom this is also to point out that — once again — we need change requests via tickets, not non-developers writing code that we have to review.

Comment (tonytlwu) [reply interface.html]: @ibroom I agree with @squallstar. We want to take code review quite seriously, and it's really best to have the changes made by developers who know how to best implement the features/changes. Otherwise time is wasted through reviewing sub-par code and requesting for justifications/changes because the person who implements the change becomes responsible for the code.

Please submit change requests via tickets.

Comment (ibroom) [reply interface.html]: I did :D https://github.com/Fliplet/fliplet-studio/issues/2387

**Comment by squallstar** [comment]
@tonytlwu I've removed the `required` attribute, this is now ready for review.

**Comment by tonytlwu** [comment]
@squallstar @ibroom We should add a check before this is submitted to ensure the following text isn't found in the review notes: `[PLEASE REVIEW AND UPDATE OR REMOVE SECTION IN SQUARE BRACKETS]`.

Otherwise, most people will just submit the apps as it is without reviewing the notes.

**Comment by squallstar** [comment]
That's a good point!

**Comment by ibroom** [comment]
When will this be deployed? We continue to have significant problems with Apple and this minor change is critical to helping it. This is a critical change.

**Comment by ibroom** [comment]
I disagree with your suggested content change, please publish. CS will accept the risk the palceholder content is left in place
